build/nogo_config.json

```diff
@@ -357,7 +357,9 @@
     },
     "only_files": {
       "//cmd/": "cmd code",
+      "lightning/": "lightning code",
       "pkg/bindinfo/": "bindinfo code",
+      "pkg/lightning/": "lightning code",
```

Comment:
is this the list of dir we have already modernized? maybe add `pkg/dist`, i see a PR have done it.

Reply:
When all gopls modernize PRs are merged, I will do a final PR to remove 'only_files' from 'intrange' nogo linter, so it also covers the ones I missed to add here.